### ISSUE TITLE
Issue with "with" in the answer

### ISSUE THREAD

**Issue by meaz** [Ajnasz/hubot-respond#5]
There is an issue if there is a "with" in the expected answer.

For example, if you do:
`hubot respond to Explain with This is my answer with a lot of sentences`

then here is what happens:
`me> Explain`
gives nothing.

```
me> Explain with This is my answer
hubot>a lot of sentences
```

So I guess only the first with should be considered with the regex, not the last one.

**Comment by Ajnasz** [comment]
Should be fixed, please check `hubot-respond@v1.2.1`

**Comment by meaz** [comment]
it is indeed! thanks
